gbpservice/neutron/tests/unit/nfp/configurator/api/v1/controllers/test_controller.py

```diff
@@ -211,21 +211,6 @@ def test_cast(self):
                                    jsonutils.dumps(self.data))
         self.assertEqual(value, True)
 
-    def test_get_notifications_fail(self):
-        """Tests failure case of HTTP get request get_notifications.
```

Comment:
This test case is to test exception case of get method. Previously we were testing it by raising exception but now we have used jsonutil instead of json which supress the exception and there are no other methods in get by which we can explicitly raise exception so, this test case is removed.